src/main/java/elements/PageElementList.java

```diff
@@ -0,0 +1,157 @@
+package elements;
+
+import factory.DriverContainer;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.*;
+
+public class PageElementList implements List<PageElement> {
```

Comment:
I thisn you should not implement list
Just implement collection and add method get(index)
And also getTexts

Reply:
fixed